app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  def markdown(text)
```

Comment:
Something like:
```ruby
##
# @param [{what types does Markdown#render accept?}] text  A markdown formatted string {(or IO object?)}
# @return [String] An HTML representation of the markdown string given in `text`
# @raise {???? does Redcarpet ever throw errors on #render or #html_safe?}
```

And a test that passes in a simple markdown string and checks that the output looks at least vaguely right; maybe also tests for any expected error conditions.